js/contact.js

```diff
@@ -1,50 +1,18 @@
-let queryName = "";
-
 // GET method route
-const getContact = (req, res) => {
-  queryName = req.query.name;
-  if (queryName == undefined) {
+const getContact = (req, res) => { 
+  let queryName = req.query.name;
+  if (queryName == undefined){
     queryName = "";
   } 
-  var html =`<form action="/contact-us" method="post"> 
-               Enter your name*: 
-               <br> 
-               <input type="text" name="name" placeholder="Full name" value=${queryName}> 
-               <br> 
-               Enter your email*: 
-               <br> 
-               <input type="email" name="emailAddress" placeholder="example@gmail.com"> 
-               <br>
-               Enter your message: 
-               <br>
-               <textarea name="message" placeholder="Your message"> </textarea>
-               <br>
-               <button>Submit</button> 
-              </form>`;
-    res.send(html);  
-}
+  res.render('contact', {name: `${queryName}`});
```

Comment:
Why not simply 
```suggestion
  res.render('contact', {name: queryName});
```
there is no need to use template literals here